pkg/prompts/prompts.go

```diff
@@ -380,3 +456,12 @@ func (*realPrompter) CaptureIndex(promptStr string, options []common.Address) (i
 	}
 	return listIndex, nil
 }
+
+func contains(list []any, element any) bool {
```

Comment:
great opportunity to use go generics?

Reply:
Already commented.

Reply:
this is another function, really contains will be much generic in that way